eng/feeds/azure-operations-scaffolding.json

```diff
@@ -0,0 +1,47 @@
+{
+  "azure-arm-operations": {
+    "title": "(rest-api-spec repo) Azure Resource Manager Service Project with Operations Resource Type",
+    "description": "Create a project with operations resource type in rest-api-spec repo, representing an Azure service ARM API",
+    "compilerVersion": "0.45.2",
+    "libraries": [],
+    "skipCompilerPackage": true,
+    "config": {
+      "emit": ["@azure-tools/typespec-autorest", "@azure-tools/typespec-autorest-canonical"]
```

Comment:
Do you need the canonical emitter for this

Reply:
removed